### PR TITLE
Content type

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* add contentType to TestFields on Address and PersonalInfoComponent

### CODE REVIEW

File: Adyen/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift [hunk 1]
```diff
@@ -89,7 +89,8 @@ open class AbstractPersonalInformationComponent: PaymentComponent, PresentableCo
         let identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "firstNameItem")
         let injector = NameFormItemInjector(identifier: identifier,
                                             localizationKey: .firstName,
-                                            style: style.textField)
+                                            style: style.textField,
+                                            contentType: .name)
```
Comment (mohammedDehairy) [inline Adyen/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift:93]: this should be `givenName`

Comment (descorp) [reply Adyen/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift:93]: 👍

**Review by mohammedDehairy — APPROVED** [review]

File: Adyen/UI/Form/Items/Address/AddressViewModel.swift [hunk 2]
```diff
@@ -82,3 +83,24 @@ internal struct AddressViewModel {
...
 }
+
+extension AddressField {
+
+    internal var contentType: UITextContentType? {
+        switch self {
+        case .street:
+            return .streetAddressLine1
+        case .houseNumberOrName:
+            return .streetAddressLine1
```
Comment (kaphacius) [inline Adyen/UI/Form/Items/Address/AddressViewModel.swift:94]: should this be `.streetAddressLine2`?

File: Adyen/UI/Form/Items/Address/AddressViewModel.swift [hunk 3]
```diff
@@ -82,3 +83,24 @@ internal struct AddressViewModel {
...
+        case .appartment:
+            return nil
+        case .postalCode:
+            return .postalCode
+        case .city:
+            return .addressCity
+        case .stateOrProvince:
+            return .addressCityAndState
```
Comment (kaphacius) [inline Adyen/UI/Form/Items/Address/AddressViewModel.swift:102]: `.addressState` maybe?

File: Adyen/UI/Form/Items/Address/AddressViewModel.swift (hunk 2 above)
Comment (descorp) [reply Adyen/UI/Form/Items/Address/AddressViewModel.swift:94]: 🤦‍♂️

File: Adyen/UI/Form/Items/Address/AddressViewModel.swift (hunk 3 above)
Comment (descorp) [reply Adyen/UI/Form/Items/Address/AddressViewModel.swift:102]: 👍

**Review by kaphacius — APPROVED** [review]

**Pull request merged**